pkgs/development/compilers/rust/rustc.nix

```diff
@@ -144,7 +161,7 @@ in stdenv.mkDerivation rec {
     ++ optionals stdenv.isDarwin [ libiconv Security ]
     ++ optional (!withBundledLLVM) llvmShared;
 
-  outputs = [ "out" "man" "doc" ];
+  outputs = [ "out" ] ++ optionals (!fast_cross) [ "man" "doc" ];
```

Comment:
As with cargo, we should make sure documentation etc is still accessible in the cross-compiler versions.